src/shared/components/palette/organisation/write-organisation-field.component.html

```diff
@@ -43,7 +43,7 @@ <h2 class="heading-h2">Organisation name and address</h2>
               <ccd-markdown [content]="organisation.address"></ccd-markdown>
             </td>
             <td class="td-select">
-              <a *ngIf="(selectedOrg$ | async).organisationIdentifier === ''" href="javascript:void(0);" (click)="selectOrg(organisation)" title="Select the organisation {{organisation.name}}">Select</a>
+              <a *ngIf="!(selectedOrg$ | async)?.organisationIdentifier" href="javascript:void(0);" (click)="selectOrg(organisation)" title="Select the organisation {{organisation.name}}">Select</a>
```

Comment:
This `*ngIf` isn't needed at all as it's nested inside of an identical one above.

Reply:
good spot, removed